README.md

```diff
@@ -189,6 +189,13 @@ jobs:
 
 **Note**: the `otp-version: false` input is only applicable when installing Gleam.
 
+## Hex.pm mirrors and caching
+
+It is possible to define the hex.pm mirrors to use including their order with
+option `hexpm-mirrors`. By default, the action will fetch from `repo.hex.pm`
+first and then from `cdn.jsdelivr.net/hex`. Permanent files from these mirrors,
+such as OTP binaries for a given version, are cached automatically.
```

Comment:
Is there a way to bust the cache?

Reply:
The cache keys depends on the MD5 sum of `setup-beam.js`.

Reply:
Is there a way for the user to bust the cache? If there is I think it would be useful to add to the documentation.

Reply:
It'll be on the consumer, right? i.e. this cache will appear as a normal cache the consumer can delete from their list of caches... If that's so we can maybe point to either something like .../actions/caches, or to GitHub Actions caches' documentation: https://docs.github.com/en/rest/actions/cache#list-github-actions-caches-for-a-repository.

Reply:
This is apparently now done via `crypto`, as per https://github.com/erlef/setup-beam/pull/156/files#diff-6406ae11684199b17d2cdca1efe2cfa29fb4587c3b142f1d33ad5f81a5acf7adR42 and subsequent lines of code...